packages/ui/src/ui/pages/groups/GroupEditorDialog/GroupEditorDialog.tsx

```diff
@@ -107,41 +122,58 @@ class GroupEditorDialog extends React.Component<GroupsPageTableProps> {
                 fields={[
                     {
                         type: 'tab-vertical',
-                        name: 'details',
-                        title: 'Details',
+                        name: 'general',
+                        title: 'General',
                         fields: [
                             {
-                                name: 'idm',
-                                type: 'plain',
-                                caption: 'Idm managed',
-                            },
-                            {
-                                name: 'size',
-                                type: 'plain',
-                                caption: 'Size',
-                            },
-                        ],
-                    },
-                    {
-                        type: 'tab-vertical',
-                        name: 'responsibles',
-                        title: 'Responsibles',
-                        fields: [
-                            {
-                                name: 'responsibles',
-                                type: 'acl-roles',
-                                caption: 'Responsibles',
-                                extras: {
-                                    placeholder: 'Enter login or name',
-                                },
-                            },
-                            {
-                                name: 'responsiblesComment',
-                                type: 'textarea',
-                                caption: 'Comment for IDM',
+                                name: 'groupName',
+                                type: 'text',
+                                required: true,
+                                caption: 'Group name',
                             },
                         ],
                     },
+                    ...(isIdmAclAvailable()
```

Comment:
Is better to use more specific options instead of `AclApi.isAllowed`. It looks like we can merge `General` and `Details` tab, but we need to hide `idm` field. We can use check if `AclApi.getGroupAcl` is defined.